src/PerfView/Extensibility.cs

```diff
@@ -3620,7 +3621,9 @@ public static MutableTraceEventStackSource ThreadTimeStacks(this TraceLog eventL
         var stackSource = new MutableTraceEventStackSource(eventLog);
         stackSource.ShowUnknownAddresses = App.CommandLineArgs.ShowUnknownAddresses;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
I don't really want lots of #pragma warning disable statements littered throughout the code.    This is a case where I think the cure is worse then the disease.      The warnings are meant to force us to fix things, and suppressing them is going in the wrong direction and making the code uglier.  

We can either
1. Fix the code not to use the Obsolete thing (may be hard)
2. If the obsolete was marking something experimental, determine if the Obsolete attribute can be removed. 
3. Make a wrapper with one #pragma warning disable in the wrapper
4. If there really is only one call site and (1) is not an option, it is OK to use #pragma warning disable. 
5. Simply disable CS0618 and log a issue to do options 1-4 and turn it back on.  
6. Just live with the warnings.  (they remind us of unfinished work).